src/main/java/org/broadinstitute/hellbender/tools/copynumber/allelic/alleliccount/AllelicCountCollection.java

```diff
@@ -0,0 +1,83 @@
+package org.broadinstitute.hellbender.tools.copynumber.allelic.alleliccount;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Simple data structure to pass and read/write a List of {@link AllelicCount} objects.
+ *
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public class AllelicCountCollection {
+    private final List<AllelicCount> counts;
+
+    public AllelicCountCollection() {
+        counts = new ArrayList<>();
+    }
+
+    /**
+     * Constructor from from file. Checks whether the input file has just the basic columns, or full columns.
```

Comment:
Oops, this is leftover copy/paste.  I think we should require that AllelicCountCollections read/written to file have all columns specified, but otherwise the ref/alt nucleotide columns can be optional (since we will typically use AllelicCountCollections to store data for model inference, where we may not care about the ref/alt nucleotides).  Will fix up code and documentation throughout to reflect this.